client/my-sites/site-settings/form-general.jsx

```diff
@@ -330,6 +330,54 @@ class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	netNeutralityOption() {
+		const { fields, isRequestingSettings, translate, handleToggle, moment, handleSubmitForm, isSavingSettings } = this.props;
+
+		const today = moment(),
+			lastDay = moment( { year: 2017, month: 6, day: 12 } );
+
+		if ( today.isAfter( lastDay, 'day' ) ) {
+			return null;
```

Comment:
That is correct. It disappears from the front-end, though so no need to explicitly turn it off.

We will be removing the UI for it and code after the event is over.